packages/core/types/index.js

```diff
@@ -1045,6 +1041,14 @@ export type Transformer<ConfigType> = {|
     options: PluginOptions,
     logger: PluginLogger,
   |}): Async<Array<TransformerResult | MutableAsset>>,
+  /** Do some processing after the transformation  */
+  postProcess?: ({|
+    assets: Array<MutableAsset>,
+    config: ConfigType,
```

Comment:
other than `config` type on this line and some comments and invalidation way is completely same to https://github.com/parcel-bundler/parcel/pull/5674/files